gf2_192/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+//! Implementation of finite field arithmetic and polynomial interpolation/evaluation in Galois
+//! field GF(2^192).
+
+// Coding conventions
+#![forbid(unsafe_code)]
+#![deny(non_upper_case_globals)]
+#![deny(non_camel_case_types)]
+#![deny(non_snake_case)]
+#![deny(unused_mut)]
+#![deny(dead_code)]
+#![deny(unused_imports)]
+#![deny(missing_docs)]
+// Clippy exclusions
+#![allow(clippy::unit_arg)]
+#![deny(rustdoc::broken_intra_doc_links)]
+#![deny(clippy::unwrap_used)]
+#![deny(clippy::expect_used)]
+#![deny(clippy::todo)]
+#![deny(clippy::unimplemented)]
+#![deny(clippy::panic)]
+
+use derive_more::From;
+use gf2_192poly::Gf2_192PolyError;
+use thiserror::Error;
+
+pub mod gf2_192;
+pub mod gf2_192poly;
+
+/// Logical right shift of i64 values which is equivalent to the `>>>` operator in java/scala. Need
+/// this because rust does 'arithmetical shift right' on signed integers.
+/// See:
+///  - https://en.wikipedia.org/wiki/Logical_shift
```

Comment:
Could you please wrap URL in `<>` to make intra-doc linter happy? It's for properly rendering link on docs.rs.

Reply:
Sure.